<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,100">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>40</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QPushButton" name="start_btn">
          <property name="text">
           <string>启动 </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stop_btn">
          <property name="text">
           <string>停止</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="hold_btn">
          <property name="text">
           <string>Hold</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="unlock_btn">
          <property name="text">
           <string>解锁</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="takeoff_btn">
          <property name="text">
           <string>起飞</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="landing_btn">
          <property name="text">
           <string>降落</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="formation">
          <item>
           <property name="text">
            <string>一字</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>人字</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>四边形</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>圆形</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="x_pos">
            <property name="text">
             <string>x_pos:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="y_pos">
            <property name="text">
             <string>y_pos:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="change_formation_btn">
          <property name="text">
           <string>变化队形</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="canvas_layout"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
